internal/test/wsu/wsu_test.go

```diff
@@ -205,6 +238,27 @@ func testWorkerLabelsArePresent(t *testing.T) {
 		"expected worker label to be present on the Windows node but did not find any")
 }
 
+// doesFileContain checks if the given file on the remote Windows virtual machine contains given string
+func doesFileContainMatchingString(fileName, matchString string) (bool, error) {
+	// Read the file from remote host and check if the hostSubnet is present
+	stdout := os.Stdout
```

Comment:
Why are you using os.Stdout here and not a new buffer like we do elsewhere in the test suite? See: https://github.com/openshift/windows-machine-config-operator/blob/master/internal/test/wsu/wsu_test.go#L135

This way you wont have to deal with all of the os.Pipe(), ReadAll(), etc.